docs/user_guide/assessment/perform_fairness_assessment.rst

```diff
@@ -199,6 +199,11 @@ These are accessed through the :attr:`MetricFrame.by_group` property:
 
 All of these values can be checked against the original arrays above.
 
+Note that :class:`MetricFrame` is intended for analyzing the differences 
+between groups with regard to a base metric, and consequently cannot take 
```

Comment:
Suggest "...analyzing the disparities between groups when evaluating a base metric, and ....."

My biggest concern is switching 'difference' for 'disparity' here

Reply:
Done